soccer/vision/filter/KalmanFilter2D.cpp

```diff
@@ -18,100 +18,83 @@ void KalmanFilter2D::createConfiguration(Configuration* cfg) {
 
 KalmanFilter2D::KalmanFilter2D() : KalmanFilter(1,1) {}
 
-KalmanFilter2D::KalmanFilter2D(Geometry2d::Point initPos, Geometry2d::Point initVel)
+KalmanFilter2D::KalmanFilter2D(geometry2d::Point initPos,
+                               geometry2d::Point initVel)
     : KalmanFilter(4, 2) {
-
     // States are X pos, X vel, Y pos, Y vel
-    x_k1_k1 << initPos.x(),
-               initVel.x(),
-               initPos.y(),
-               initVel.y();
+    x_k1_k1 << initPos.x(), initVel.x(), initPos.y(), initVel.y();
     x_k_k1 = x_k1_k1;
     x_k_k = x_k1_k1;
 
-    // Initial covariance is usually extremely high to converge to the true solution
+    // Initial covariance is usually extremely high to converge to the true
+    // solution
     double p = *ball_init_covariance;
-    P_k1_k1 << p, 0, 0, 0,
-               0, p, 0, 0,
-               0, 0, p, 0,
-               0, 0, 0, p;
+    P_k1_k1 << p, 0, 0, 0, 0, p, 0, 0, 0, 0, p, 0, 0, 0, 0, p;
```

Comment:
The matrix initialization in here should be exempt from style IMO. They were lined up to match the matrix size